### ISSUE TITLE
Ansible as Teletype expander ?

### ISSUE THREAD

**Issue by davephillips** [Dewb/monome-rack#125]
According to the Ansible docs the module can act as an expander for Teletype if nothing else is connected, so something like "TR.P 5" should send a trigger from Teletype to the first trigger output from Ansible. Alas, it's not working here. Is it supposed to ? Also, is the Mode indicator light supposed to be functional ? I'm building from latest source as usual.

Btw, this isn't a big issue for me, everything's working nicely otherwise. Meadowphysics is a blast. :)

**Comment by Dewb** [comment]
From the 2.0.2-beta release notes:
> Not currently supported: the MIDI and "teletype expander" modes from the hardware

This is something I want to tackle eventually but it depends on #15.

**Comment by Dewb** [comment]
And yes, the mode indicator light works, but you need to connect a grid or arc before mode selection is active.

**Comment by davephillips** [comment]
Ah, thanks a lot ! What I get for not reading the notes. :) Sorry for the noise.

**Comment by Dewb** [comment]
No worries, thanks for the feedback!

**Comment by Dewb** [comment]
Duplicate of #65
